MDANSE/Src/MDANSE/Framework/Jobs/DistanceHistogram.py

```diff
@@ -156,32 +158,33 @@ def detailed_unit_cell_error(self):
         )
 
     def run_step(self, index):
-        """
-        Runs a single step of the job.\n
-
-        :Parameters:
-            #. index (int): The index of the step.
-        :Returns:
-            #. index (int): The index of the step.
-            #. cellVolume (float): the volume of the current frame simulation box
-            #. hIntraTemp (np.array): The calculated distance intra-molecular histogram
-            #. hInterTemp (np.array): The calculated distance inter-molecular histogram
-        """
+        """Run a single step of the analysis.
+
+        Parameters
+        ----------
+        index : int
+            number of the simulation frame for which to calculate the distances.
 
+        Returns
+        -------
+        int, tuple[float, np.ndarray, np.ndarray]
+            repeated input index and the analysis results
```

Comment:
Numpydoc spec allows this spec to be:
```suggestion
        int
            Repeated input index.
        tuple[float, ~numpy.ndarray, ~numpy.ndarray]
            The analysis results.
```